DigitalLearningSolutions.Data/Models/User/DelegateUser.cs

```diff
@@ -46,5 +48,12 @@ public CentreAnswersData GetCentreAnswersData()
                 Answer6
             );
         }
+
+        public string GetPrnDisplayString()

```

Comment:
Worth putting this as an expression body definition property and making some unit tests for it?

Reply:
I'd probably put this in DisplayStringHelper

Reply:
Done!

Reply:
Ah whoops, I must have not refreshed the page and missed the comment about DisplayStringHelper. Moved the method to there.